components/Table/Table.jsx

```diff
@@ -36,13 +37,17 @@ const getTable = (data, columns, sortState, sortField, sortClick, numRows) => {
         </MaterialTableHeader>
         <MaterialTableBody displayRowCheckbox={false} selectable={false}>
           {modifiedData.map((row, index) => (
-            <TableRow
+            <MaterialTableRow
               key={index}
-              row={row}
-              columns={columns}
-              totalWidth={totalWidth}
-              index={index}
-            />
+              style={{ borderBottomWidth: '0px' }}
+            >
+              {columns.map((column, colIndex) => (
+                <MaterialTableRowColumn key={colIndex} style={getWidthStyle(column.width, totalWidth)}>
+                  {row && column.displayFn && column.displayFn({ row, column, field: row[column.field] })}
+                  {row && row[column.field] && !column.displayFn && (row[column.field].display)}
+                </MaterialTableRowColumn>
+              ))}
+            </MaterialTableRow>
           ))}
```

Comment:
* It was breaking the ability to pass the material-ui props down (workaround is to add them into the wrapper component, but that seems unnecessarily verbose)
* It's easier to follow what's going on

Did we need to componentize for a specific reason?